.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Details
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide any publicly-available historical context._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any publicly-available hardware peculiarities._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
+
+**Test Cases:**
+- [ ] _List here_
```

Comment:
For bug fix PRs, test cases are not straightforward most of time, i.e. need framework dockers, special test procedure and exact gfx/machines, etc. Again, better reference to JIRA or github issues for details

Reply:
I too would recommend removing this section.

Reply:
Removing this.